README.md

```diff
@@ -19,11 +19,11 @@ import {
 } from 'zxcvbn-custom-matchers';
 
 // Add the matchers
-zxcvbnOptions.addMatcher('lowercase', lowercaseMatcher);
-zxcvbnOptions.addMatcher('number', numberMatcher);
-zxcvbnOptions.addMatcher('special', specialMatcher);
-zxcvbnOptions.addMatcher('uppercase', uppercaseMatcher);
-zxcvbnOptions.addMatcher('minLength', minLengthMatcher(10));
+zxcvbnOptions.addMatcher('oneLowercase', lowercaseMatcher);
```

Comment:
i'd opt for using `lowercaseRequired`, `numberRequired`, `specialCharRequired` and etc. instead.

Reply:
Need to also update the readme to use the correct names for the matchers.

Reply:
already updated